src/js/control-bar/time-controls/time-display.js

```diff
@@ -94,7 +94,7 @@ class TimeDisplay extends Component {
         return;
       }
 
-      if (oldNode) {
```

Comment:
can we add a special check for whether contentEl_ contains oldNode so that we can log a warning?